### PR TITLE
feat: fallback data for pagehome regular projects

### PR DESCRIPTION
**Pull request by sk-umiuma**
image

Resolves: https://app.asana.com/0/0/1204062700178987/f

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -41,23 +41,40 @@ export const getServerSideProps: GetServerSideProps<Props> = async (
...
+    getAllProjects(db, featuredProjects$Params).catch((error) => {
+      console.error("[SSR | Home | featuredProjects]", error);
+      return undefined;
+    }),
+    getAllProjects(db, sponsoredProjects$Params).catch((error) => {
+      console.error("[SSR | Home | sponsoredProjects]", error);
+      return undefined;
+    }),
+    getAllProjects(db, regularProjects$FirstPage$Params).catch((error) => {
+      console.error("[SSR | Home | regularProjects]", error);
```
Comment (sk-kitsune) [inline pages/index.tsx:67]: for sake of consistency, return `undefined` here

**Comment by sk-kitsune** [comment]
Please rebase.

**Comment by sk-kitsune** [comment]
Please provide a proof that Google Bot can see the projects on the first page.

See https://www.notion.so/shinka-network/Guidelines-on-improving-SEO-77be08130bf1426891aaff68c69a8339?pvs=4#7e1fa392560c44ee90f47933fbe4c256

**Review by sk-kitsune — CHANGES_REQUESTED** [review]
⛔ **Changes requested**: there are blocking issues.

**Comment by sk-shishi** [comment]
@sk-kitsune Umiuma will send you the proof later xD

**Comment by sk-kitsune** [comment]
Ok, sure

**Pull request merged**